versions.yaml

```diff
@@ -191,6 +188,11 @@ assets:
     url: "https://github.com/kata-containers/linux/archive/refs/tags"
     version: "5.19-TDX-v2.2"
 
+  kernel-sev-experimental:
+    description: "Linux kernel that supports SEV"
+    url: "https://cdn.kernel.org/pub/linux/kernel/v5.x/"
+    version: "v5.19.2"
```

Comment:
So, it doesn't need to be an experimental one.

Reply:
I find these changes confusing, why is `sev` removed, and why is it `sev-experimental` and not `snp-experimental`?